tests/test_fips.py

```diff
@@ -320,3 +320,35 @@ def test_icainfo_binary(container_per_test: ContainerData) -> None:
     )
     container_per_test.connection.check_output("icastats -k")
     container_per_test.connection.check_output("icastats -S")
+
+
+@pytest.mark.parametrize(
+    "container_per_test", FIPS_TESTER_IMAGES, indirect=True
+)
+def test_nss_firefox_cert(container_per_test: ContainerData) -> None:
+    container_per_test.connection.check_output(
+        "zypper -n install mozilla-nss-tools"
+    )
+    # Seed file
+    container_per_test.connection.check_output(
+        "dd if=/dev/urandom bs=1k count=32767 of=seedfile.dat"
+    )
+    # Empty password
+    container_per_test.connection.check_output("touch password.txt")
+    # Create database, needed prerequisite, needs one <enter> as interaction
+    assert (
+        container_per_test.connection.check_output(
+            'rm -rf nssdb && mkdir -p nssdb && certutil -N -d "${PWD}/nssdb" --empty-password && modutil -fips true -dbdir "${PWD}/nssdb" -force'
```

Comment:
No need to remove a non-existent directory
```suggestion
            'mkdir -p nssdb && certutil -N -d "${PWD}/nssdb" --empty-password && modutil -fips true -dbdir "${PWD}/nssdb" -force'
```

Reply:
Changed :heavy_check_mark: